source/stylesheets/overview.scss

```diff
@@ -6,7 +6,7 @@
   border: none;
   height: 53px;
   padding-top: 7px;
-  font-size: 18px;
+  font-size: calc(22px + 0.3vw);
```

Comment:
Can you explain what's going on here?

Reply:
I can! The font-size is being calculated (`calc()`) rather than a fixed single value. There are lots of other examples of this kind of calculation in other varnish stylesheets so I'm not sure why this wasn't done here. In any case, the calc function uses a pixel-based value and a viewport-based ratio to set the font size. The viewport width should be the immediate parent container's width IIRC.